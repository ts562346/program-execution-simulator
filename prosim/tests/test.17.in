test1 14
LOOP 4
  DOOP 2
  LOOP 3
    BLOCK 3
    DOOP 1
  END
  BLOCK 5
  LOOP 5
    BLOCK 1
    DOOP 9
  END
  BLOCK 5
END
HALT

