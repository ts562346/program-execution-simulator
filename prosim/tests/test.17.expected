00000: DOOP
00002: BLOCK
00005: DOOP
00006: BLOCK
00009: DOOP
00010: BLOCK
00013: DOOP
00014: BLOCK
00019: BLOCK
00020: DOOP
00029: BLOCK
00030: DOOP
00039: BLOCK
00040: DOOP
00049: BLOCK
00050: DOOP
00059: BLOCK
00060: DOOP
00069: BLOCK
00074: DOOP
00076: BLOCK
00079: DOOP
00080: BLOCK
00083: DOOP
00084: BLOCK
00087: DOOP
00088: BLOCK
00093: BLOCK
00094: DOOP
00103: BLOCK
00104: DOOP
00113: BLOCK
00114: DOOP
00123: BLOCK
00124: DOOP
00133: BLOCK
00134: DOOP
00143: BLOCK
00148: DOOP
00150: BLOCK
00153: DOOP
00154: BLOCK
00157: DOOP
00158: BLOCK
00161: DOOP
00162: BLOCK
00167: BLOCK
00168: DOOP
00177: BLOCK
00178: DOOP
00187: BLOCK
00188: DOOP
00197: BLOCK
00198: DOOP
00207: BLOCK
00208: DOOP
00217: BLOCK
00222: DOOP
00224: BLOCK
00227: DOOP
00228: BLOCK
00231: DOOP
00232: BLOCK
00235: DOOP
00236: BLOCK
00241: BLOCK
00242: DOOP
00251: BLOCK
00252: DOOP
00261: BLOCK
00262: DOOP
00271: BLOCK
00272: DOOP
00281: BLOCK
00282: DOOP
00291: BLOCK
00296: HALT

DOOP count : 36
DOOP time  : 200
BLOCK count: 40
BLOCK time : 96
